src/scripts/main.js

```diff
@@ -1,5 +1,27 @@
 'use strict';
 
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+const wallHight = wall.clientHeight;
```

Comment:
The variable name `wallHight` is misspelled. It should be `wallHeight` to accurately reflect its purpose.